django/contrib/auth/views.py

```diff
@@ -200,6 +201,10 @@ def password_reset(request, is_admin_site=False,
                     RemovedInDjango110Warning, 3
                 )
                 opts = dict(opts, domain_override=request.get_host())
+
+            if extra_email_context is not None:
+                opts.update({'extra_email_context': extra_email_context})
```

Comment:
Hey Tim,
I know that extra_email_context will be a dictionary. But before we can
merge that dictionary with opts, we need to know the keywords inside
extra_email_context as forms.save() expects the keywords as parameters in
password_reset_form. Hence, I am just appending a keyword to the dictionary
sent by the user so that form.save() can accept any dictionary sent by the
user irrespective of the keywords defined by the user.
For example, assuming that user has defined extra_email_context to be {
Keyword1 : Value1 , Keyword2 : Value2 } , form.save() need to know Keyword1
and Keyword2. Therefore, I am converting it into { extra_email_context :  {
Keyword1 : Value1 , Keyword2 : Value2 }  }
Is this ok? Thanks!

On Thu, Sep 10, 2015 at 5:36 AM, Tim Graham <notifications@github.com>
wrote:

> In django/contrib/auth/views.py
> <https://github.com/django/django/pull/5190#discussion_r39112446>:
>
> > @@ -200,6 +201,10 @@ def password_reset(request, is_admin_site=False,
> >                      RemovedInDjango110Warning, 3
> >                  )
> >                  opts = dict(opts, domain_override=request.get_host())
> > +
> > +            if extra_email_context is not None:
> > +                opts.update({'extra_email_context': extra_email_context})
>
> opts.update(extra_email_context) (since extra_email_context will be a
> dictionary)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5190/files#r39112446>.
>